packages/query/src/block-processor.ts

```diff
@@ -162,12 +163,18 @@ export class BlockProcessor implements BlockProcessorInterface {
         await this.indexedDb.saveFmdParams(compactBlock.fmdParameters);
       }
       if (compactBlock.gasPrices) {
-        // TODO #1310 pre-populate assetId for native GasPrices using stakingTokenAssetId
-        await this.indexedDb.saveGasPrices(compactBlock.gasPrices);
+        await this.indexedDb.saveGasPrices(
+          new GasPrices({
+            assetId: this.stakingAssetId,
+            ...compactBlock.gasPrices,
+          }),
+        );
+      }
+      if (compactBlock.altGasPrices.length) {
+        for (const gasPrice of compactBlock.altGasPrices) {
+          await this.indexedDb.saveGasPrices(gasPrice);
+        }
```

Comment:
this suggests that the RPC node is returning information in compact blocks about _two_ alternative gas fees, but with zero gas prices. Manual testing confirmed that all _other_ token denominations result in a "GasPrices not available" error.

<img width="1675" alt="Screenshot 2024-07-04 at 1 15 13 AM" src="https://github.com/penumbra-zone/web/assets/70081547/82fb68e0-7e71-4e43-9c01-53f369018494">